backend/attendance-service/src/main/java/org/egov/enrichment/AttendanceLogEnrichment.java

```diff
@@ -0,0 +1,72 @@
+package org.egov.enrichment;
+
+import digit.models.coremodels.AuditDetails;
+import lombok.extern.slf4j.Slf4j;
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.config.AttendanceServiceConfiguration;
+import org.egov.web.models.AttendanceLogSearchCriteria;
+import org.egov.web.models.Document;
+import org.egov.web.models.AttendanceLog;
+import org.egov.web.models.AttendanceLogRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.UUID;
+
+@Component
+@Slf4j
+public class AttendanceLogEnrichment {
+
+    @Autowired
+    private AttendanceServiceConfiguration config;
+    public void enrichAttendanceLogCreateRequest(AttendanceLogRequest attendanceLogRequest) {
+        List<AttendanceLog> attendanceLog = attendanceLogRequest.getAttendance();
+        AuditDetails auditDetails = getAuditDetails(attendanceLogRequest);
```

Comment:
You need to setAuditDetails for the attendanceLog object.

Reply:
We are setting auditDetails on AttendanceLog object on line no 30. Inside the loop.